claude/commands/CLAUDE.md

```
@@ -0,0 +1,12 @@
+<claude-mem-context>
+# Recent Activity
+
+<!-- This section is auto-generated by claude-mem. Edit content outside the tags. -->
+
+### Jan 27, 2026
+
+| ID | Time | T | Title | Read |
+|----|------|---|-------|------|
+| #4289 | 8:47 PM | 🟣 | Created Claude-Compatible OpenSpec Review Command | ~354 |
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)